### PR TITLE
update architecture image

### PR DESCRIPTION
**Pull request by yyrdl**
**What type of PR is this?**

update the architecture image ,add estimator 

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE

```

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 .idea/
 .vscode/
+.DS_Store
```
Comment (RainbowMango) [inline .gitignore:19]: What's the difference between `.DS_Store` and `.DS_Store/`?  @XiShanYongYe-Chang

**Comment by RainbowMango** [comment]
Hi @yyrdl ,

The `karmada-estimator` usually not running in member clusters, so I'm afraid this will mislead people.

In addition, I think the `karmada-estimator` is a sub-component of `karmada-scheduler`, in case people get confused, I'd prefer to keep architecture simple and add some descriptions to docs somewhere.

I'm ok with the gitignore part.

**Comment by yyrdl** [comment]
Emm，if it's just a component in scheduler , it's ok ,but it is a service .  "keep architecture simple and add some descriptions to docs somewhere" I agree with this. we can explain how the scheduler works somewhere else.

**Pull request closed without merging**